docs/setup/settings.asciidoc

```diff
@@ -50,6 +50,9 @@ error messages.
 information and all requests.
 `logging.useUTC`:: *Default: true* Set the value of this setting to `false` to log events using the timezone of the server, rather than UTC.
 
+`map.includeElasticMapsService:`:: turns on or off whether layers from the Elastic Maps Service should be included in the vector and tile layer option list.
```

Comment:
How about defining the default to be consistent with other options? I also played around with the wording.

> *Default: true* Toggles Elastic Maps Service layers from being included in the vector and tile layer options list. By disabling, only the layers that are configured here will be included.

Reply:
:+1: